docs/parameters.yaml

```diff
@@ -30,6 +30,23 @@ default: "~/.config/pelican"
 components: ["*"]
 type: filename
 ---
+name: ConfigLocations
+description: |+
+  `ConfigLocations` provides administrators a way to define a list of directories containing Pelican configuration files. Within a given directory,
+  files are read in lexicographical order, and any keys that are defined in multiple files will take the value from the last file read. Directories are
+  read in the order provided by the list. For example, specifying:
+
+  `ConfigLocations: ["/configs1", "/configs2"]`
```

Comment:
Are we still handling recursion here? Would it worth mentioned the behavior of nested `ConfigLocations`?